plugins/postcss-contrast-color-function/test/basic.preserve-false.expect.css

```diff
@@ -0,0 +1,169 @@
+.dynamic {
+	color: color(display-p3 0.15472 0 0.15897);
+}@media (prefers-contrast: more) {.dynamic {
+	color: rgb(0, 0, 0);
+}
+}@media (prefers-contrast: less) {.dynamic {
+	color: color(display-p3 0.29314 0 0.30019);
+}
+}
+
+.max {
+	color: rgb(0, 0, 0);
+}
+
+.both {
+	color: color-mix(in srgb, rgb(255, 255, 255), color(display-p3 0 0.10723 0.10958));
+}
+
+@media (prefers-contrast: more) {
+
+.both {
+	color: color-mix(in srgb, rgb(255, 255, 255), rgb(0, 0, 0));
+}
+}
+
+@media (prefers-contrast: less) {
+
+.both {
+	color: color-mix(in srgb, rgb(255, 255, 255), color(display-p3 0 0.21431 0.2182));
+}
+}
+
+
+.dynamic-green {
+	color: color(display-p3 0.82734 1 0.78138);
+}
+
+
+@media (prefers-contrast: more) {
+
+
+.dynamic-green {
+	color: rgb(255, 255, 255);
+}
+}
+
+
+@media (prefers-contrast: less) {
+
+
+.dynamic-green {
+	color: rgb(255, 255, 255);
+}
+}
+
+.max-green {
+	color: rgb(255, 255, 255);
+}
+
+.dynamic-lime {
+	color: color(display-p3 0.01598 0.11486 0);
+}
+
+@media (prefers-contrast: more) {
+
+.dynamic-lime {
+	color: rgb(0, 0, 0);
+}
+}
+
+@media (prefers-contrast: less) {
+
+.dynamic-lime {
+	color: color(display-p3 0.05256 0.22698 0);
+}
+}
+
+.max-lime {
+	color: rgb(0, 0, 0);
+}
+
+.dynamic-darkgreen {
+	color: color(display-p3 0.82734 1 0.78138);
+}
+
+@media (prefers-contrast: more) {
+
+.dynamic-darkgreen {
+	color: rgb(255, 255, 255);
+}
+}
+
+@media (prefers-contrast: less) {
+
+.dynamic-darkgreen {
+	color: color(display-p3 0.71003 0.96022 0.64341);
+}
+}
+
+.max-darkgreen {
+	color: rgb(255, 255, 255);
+}
+
+.manual-fallback {
+	color: white;
+	color: rgb(255, 255, 255);
+}
```

Comment:
Without the change to index.ts, this would read

```css
.manual-fallback {
	color: white;
	color: contrast-color(darkgreen max);
}
```

which would be quite an unexpected result, as it effectively skips processing.